src/content/eth2/merge/index.md

```diff
@@ -4,29 +4,29 @@ description: Learn about the merge - when mainnet Ethereum joins the Beacon Chai
 lang: en
 template: eth2
 sidebar: true
-image: ../../../assets/eth2/docking.png
+image: ../../../assets/eth2/merge.png
 summaryPoints:
   [
     'Eventually the current Ethereum mainnet will "merge" with the beacon chain proof-of-stake system.',
-    "This will mark the end of proof-of-work for Ethereum, and the full transition to proof of stake.",
-    'You might know this as "Phase 1.5" on technical roadmaps.',
-    'We formerly referred to this as "the docking.',
+    "This will mark the end of proof-of-work for Ethereum, and the full transition to proof-of-stake.",
+    "This is planned to preceed the roll out of shard chains.",
+    'We formerly referred to this as "the docking."',
   ]
 ---
 
 <UpgradeStatus date="~2021/22">
-    This upgrade will follow the arrival of shard chains. But it’s the moment where the <a href="/eth2/vision/">Eth2 vision</a> becomes fully realised – more scalability, security, and sustainability with staking supporting the whole network.
+    This upgrade is currently planned to follow the London upgrade, which is coming in the summer of 2021. It represents the disabling of proof-of-work mining, and upgrades the Ethereum you know and love today to a network secured by staked ether. An exciting step in realizing the <a href="/eth2/vision/">Eth2 vision</a> – more scalability, security, and sustainability.
```

Comment:
I feel like mentioning the London upgrade just poses more questions we don't answer at this point.

Reply:
I find this second sentence quite hard to read, too many things going on. Can we simplify or break into some separate sentences?

Reply:
Agreed, no need to mention London here.